### ISSUE TITLE
Can the wiki be displayed directly or indirectly on the homepage menu?

### ISSUE THREAD

**Issue by youngledo** [canonical/ubuntu.com#14461]
### Summary

Many times I want to know the code names of different versions of ubuntu, but I found that the homepage is not directly or indirectly displayed. It was not until the stackoverflow that I found this address: [https://wiki.ubuntu.com/](https://wiki.ubuntu.com/)

### Steps to reproduce the behavior

none, ignore.

### Expected behavior

the wiki be displayed directly or indirectly on the homepage menu.

### Browser/device details

- wiki
<img width="1248" alt="image" src="https://github.com/user-attachments/assets/bee652f2-f5dd-404c-8802-9ac3acb8d981">

- wiki/release
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/2816fd85-b04b-49b7-954c-6e4ae84c26db">

### Reported from

GitHub

**Comment by youngledo** [comment]
In addition, the website page is mixed in Chinese and English, can it be corrected and beautified?

**Comment by mtruj013** [comment]
Thanks @youngledo, to find more information regarding past releases on ubuntu.com please visit https://ubuntu.com/about/release-cycle. Regarding changes to the wiki, this is out of scope for us as this repo deals only with issues pertaining to the ubuntu.com website specifically.

**Comment by youngledo** [comment]
Thanks, although incomplete, it can be used.
